src/Stripe.net/Entities/_base/StripeEntity.cs

```diff
@@ -4,11 +4,46 @@ namespace Stripe
     using System.Reflection;
     using System.Runtime.CompilerServices;
     using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
     using Stripe.Infrastructure;
 
     [JsonObject(MemberSerialization.OptIn)]
     public abstract class StripeEntity : IStripeEntity
     {
+        [JsonIgnore]
+        private JObject rawJObject;
+
+        /// <summary>
+        /// Gets the raw <see cref="JObject">JObject</see> exposed by the Newtonsoft.Json library.
+        /// This can be used to access properties that are not directly exposed by Stripe's .NET
+        /// library.
+        /// </summary>
+        /// <remarks>
+        /// You should always prefer using the standard property accessors whenever possible. This
+        /// accessor is not considered fully stable and might change or be removed in future
+        /// versions.
+        /// </remarks>
+        /// <returns>The raw <see cref="JObject">JObject</see>.</returns>
+        [JsonIgnore]
+        public JObject RawJObject
+        {
+            get
+            {
+                // Lazily initialize the object the first time the getter is called.
+                if (this.rawJObject == null)
+                {
+                    this.rawJObject = JObject.Parse(this.StripeResponse.Content);
```

Comment:
Now that I think about it, why don't we just tell users to do that themselves?

Reply:
We could. It's literally one line of code, and `JObject` is very easy to use. If you want to close this PR and just advise users to use that line of code instead, that's fine by me!

Reply:
Hmmm I guess us deserializing means we keep using the correct settings internally. Let's ship this since it's easier to discover I think.

Reply:
shouldn't you check that the `StripeResponse` is not null?